app/service/callback_rest.py

```diff
@@ -28,146 +28,93 @@
 
 
 @service_callback_blueprint.route("/inbound-api", methods=["POST"])
-def create_service_inbound_api(service_id):
+@service_callback_blueprint.route("/delivery-receipt-api", methods=["POST"])
+@service_callback_blueprint.route("/returned-letter-api", methods=["POST"])
```

Comment:
You can add the new, generic URLs as part of this PR, rather than needing to do it later in a separate PR and deploy:
```suggestion
@service_callback_blueprint.route("/returned-letter-api", methods=["POST"])
@service_callback_blueprint.route("/callback", methods=["POST"])

Same comment applies for all endpoints in this PR.
```

Reply:
I can roll this out as part of the callback tables merge process

Reply:
Now that https://github.com/alphagov/notifications-admin/pull/5417  and https://github.com/alphagov/notifications-api/pull/4405 have been merged, I will add the new generic URLs as part of this PR.